tests/BaseTestCase.php

```diff
@@ -21,8 +21,9 @@
 use OpenCFP\Test\Helper\RefreshDatabase;
 use Pimple\Psr11\Container;
 use Psr\Container\ContainerInterface;
+use Silex\WebTestCase;
 
-abstract class BaseTestCase extends \PHPUnit\Framework\TestCase
+abstract class BaseTestCase extends WebTestCase
```

Comment:
@derrabus 

What do you think about getting rid of the `BaseTestCase` entirely?

Currently, we have `OpenCFP\Test\Integration\WebTestCase` which extends from `OpenCFP\Test\BaseTestCase`. If we now have the `BaseTestCase` extend the `Silex\WebTestCase`, I don't really see any use for the `BaseTestCase` at all.

What do you think?

Reply:
If that means that all unit tests will extend PHPUnit's test case class directly, go for it. If that means, some unit tests are switched to `WebTestCase`, I'm not really sure. That class contains a toolset that is not meant for unit tests.

Reply:
> If that means that all unit tests will extend PHPUnit's test case class directly, go for it. 

👍